packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -340,16 +345,24 @@ const AccountTransferForm = observer(
                 return internal_remaining_transfers?.available;
             };
 
-            remaining_transfers = getRemainingTransfers();
-            has_reached_maximum_daily_transfers = !Number(remaining_transfers);
-
-            const hint =
-                remaining_transfers && Number(remaining_transfers) === 1
-                    ? localize('You have {{number}} transfer remaining for today.', { number: remaining_transfers })
-                    : localize('You have {{number}} transfers remaining for today.', { number: remaining_transfers });
-            setTransferToHint(hint);
+            remaining_transfers = Number(getRemainingTransfers() ?? 0);
+            has_reached_maximum_daily_transfers = !remaining_transfers;
+
+            let hint_text;
+            if (is_migration_status_present) {
+                hint_text = <Localize i18n_default_text='You can no longer open new positions with this account.' />;
+            } else {
+                const transfer_text = remaining_transfers > 1 ? 'transfers' : 'transfer';
+                hint_text = (
+                    <Localize
+                        i18n_default_text='You have {{remaining_transfers}} {{transfer_text}} remaining for today.'
+                        values={{ remaining_transfers, transfer_text }}
+                    />
+                );
+            }
+            setTransferToHint(hint_text);
             resetConverter();
-        }, [selected_to, selected_from, account_limits]); // eslint-disable-line react-hooks/exhaustive-deps
+        }, [account_limits, is_migration_status_present, selected_from, selected_to]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
```suggestion
        }, [account_limits, is_migration_status_present, selected_from, selected_to]);
```
Most of the time this react exhaustive deps produces a valid warning. Can you remove this eslint ignore and update the deps?